rosa_cluster_admin/rosa-cluster-notifications.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-cluster-notifications"]
+= Cluster notifications
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-cluster-notifications
+
+toc::[]
+
+include::snippets/managed-openshift-about-cluster-notifications.adoc[leveloffset=+0]
+
+[role="_additional-resources"]
+== Additional resources
+* xref:../rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.adoc#notifications_rosa-policy-responsibility-matrix[Customer responsibilities: Review and action cluster notifications]
+* xref:../rosa-cluster-notifications.adoc#cluster-notification-emails_rosa-cluster-notifications[Cluster notification emails]
+* xref:../rosa-cluster-notifications.adoc#troubleshoot_rosa-cluster-notifications[Troubleshooting: Cluster notifications]
```

Comment:
This link does not work.

Reply:
This is a self-referencing XREF problem, just checking if our build scripts will accept a parentless anchor or if I need to remove this entirely.

Reply:
Confirmed a basic anchor works for these - leaving this comment open as a note for peer reviewers that this is the reason these few xrefs do not follow typical `xref:<file>#anchor` convention.